napari/_app_model/_submenus.py

```diff
@@ -30,4 +30,57 @@
         MenuId.MENUBAR_VIEW,
         SubmenuItem(submenu=MenuId.VIEW_SCALEBAR, title=trans._('Scale Bar')),
     ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(
+            submenu=MenuId.TOOLS_ACQUISITION, title=trans._('Acquisition')
+        ),
+    ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(submenu=MenuId.TOOLS_FILTERS, title=trans._('Filters')),
+    ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(
+            submenu=MenuId.TOOLS_TRANSFORM, title=trans._('Transform')
+        ),
+    ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(
+            submenu=MenuId.TOOLS_UTILITIES, title=trans._('Utilities')
+        ),
+    ),
```

Comment:
If the order is relevant, I would put Utilities last

Reply:
done!